queue_services/account-mailer/src/account_mailer/worker.py

```diff
@@ -197,6 +197,58 @@ async def process_event(event_message: dict, flask_app):
                 template_name=template_name,
                 subject=subject, **email_msg)
 
+        else:
+            template_name, subject = map_templates_based_on_message_type(message_type)
+
+            if template_name is None or subject is None:
+                return
+
+            if message_type in (MessageType.ADMIN_NOTIFICATION.value,
```

Comment:
what if we merge all the dict ? no Ifs ..just put all the values in kwargs and pass it? if the template doesnt need , it wont fail rite?